charts/sigstore-prober/values.yaml

```diff
@@ -6,7 +6,7 @@ serviceAccount:
   create: false
 spec:
   replicaCount: 1
-  image: ghcr.io/sigstore/scaffolding/prober@sha256:1d12dfb24baa55339fa6ca6f40836c39aba6ffc407ebd0b611670c6f00738cc4  # v0.4.9
+  image: ghcr.io/sigstore/scaffolding/prober@sha256:101a6a79e25944797a2ae06c2b330995995e32a35b8616db1400206e69d2a340  # v0.4.10
   imagePullPolicy: Always
   matchLabels:
```

Comment:
could you add in an empty `rekorRequests` and `fulcioRequests` in this file as the default? I think you'll also need to update the [README.md](https://github.com/sigstore/helm-charts/blob/main/charts/sigstore-prober/README.md) and [values.schema.json](https://github.com/sigstore/helm-charts/blob/main/charts/sigstore-prober/values.schema.json) files with the new values!